atc/db/migration/migrations/1626194317_add_pipeline_pauses_table.up.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE pipeline_pauses (
+  pipeline_id integer PRIMARY KEY,
+  paused boolean NOT NULL DEFAULT TRUE,
```

Comment:
since we are going to keep the column in the original table, we won't need this attribute anymore